files/solr_check_instance.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -eu
+# Step through all cores under one instance and throw an exit code if something is not running.
+# Exit code 5: hard fail, no need to wait any more
+# Exit code 2: possibly not loaded yet, try waiting a bit
+
+set -o pipefail
```

Comment:
why not adding to the shebang?

Reply:
Not working this way
```
root@ip-10-13-6-84:~# /usr/local/bin/solr_check_instance.sh mysite
/bin/bash: line 0: /bin/bash: /usr/local/bin/solr_check_instance.sh: invalid option name
```
with
`#!/bin/bash -euo pipefail`

Reply:
well. i.. so weird. maybe we should start using `set ` for all options then, instead of relying that the script is always called as an executable. context, some scp scripts were called with base script.sh and the shebang got ignored.

Reply:
yeah, I'll move everything into set